Content/RootScript/BaseClasses/Entities/ItemDefinitions/FirstAid.as

```diff
@@ -0,0 +1,269 @@
+// First Aid --------------------------------------
+
+class ItemTemplate_FirstAid : Item::Template {
+	Item@ MakeInstance() { Item @instance = Item_FirstAid(@this); return @instance; }
+	ItemTemplate_FirstAid() { super();
+		name = "FirstAid"; //"firstaid"
+		modelPath = "SCPCB/GFX/Items/Firstaid/firstaid.fbx";
+		modelSkin = "SCPCB/GFX/Items/Firstaid/firstaid.jpg";
+		modelScale = 0.5;
+
+		pickSoundID=1;
+
+		useModelIcon=true;
+		iconImage = "SCPCB/GFX/Items/Firstaid/inv_firstaid.jpg";
+		iconScale = 0.1;
+		iconRot = Vector3f(-2.3,-0.3,0.2);
+		iconPos = Vector2f(0,0.05);
+		Item::Register(@this);
+	}
+}
+ItemTemplate_FirstAid OriginItem_FirstAid;
+
+
+shared class Item_FirstAid : Item {
+	Item_FirstAid(Item::Template@&in origin) { super(@origin);
+		// onConstructed();
+	}
+
+	void Test() {
+		Debug::log("Im a FirstAid");
+	}
+	void Localize(Item@&in instance) {
+	}
+}
+
+// Small First Aid --------------------------------------
+
+class ItemTemplate_SmallFirstAid : Item::Template {
+	Item@ MakeInstance() { Item @instance = Item_SmallFirstAid(@this); return @instance; }
+	ItemTemplate_SmallFirstAid() { super();
+
+		name = "SmallFirstAid"; //"finefirstaid"
+		modelPath = "SCPCB/GFX/Items/Firstaid/firstaid.fbx";
+		modelSkin = "SCPCB/GFX/Items/Firstaid/firstaid.jpg";
+		modelScale = 0.3;
+
+		pickSoundID=1;
+
+		useModelIcon=true;
+		iconImage = "SCPCB/GFX/Items/Firstaid/inv_firstaid_blue.jpg";
+		iconScale = 0.1;
+		iconRot = Vector3f(-2.3,-0.3,0.2);
+		iconPos = Vector2f(0,0.05);
+		Item::Register(@this);
+	}
+}
+ItemTemplate_SmallFirstAid OriginItem_SmallFirstAid;
+
+
+shared class Item_SmallFirstAid : Item {
+	Item_SmallFirstAid(Item::Template@&in origin) { super(@origin);
+		// onConstructed();
+	}
+
+	void Test() {
+		Debug::log("Im a SmallFirstAid");
+	}
+	void Localize(Item@&in instance) {
+	}
+}
+
+// Blue First Aid --------------------------------------
+
+class ItemTemplate_BlueFirstAid : Item::Template {
+	Item@ MakeInstance() { Item @instance = Item_BlueFirstAid(@this); return @instance; }
+	ItemTemplate_BlueFirstAid() { super();
+
+		name = "BlueFirstAid"; //"firstaid2"
+		modelPath = "SCPCB/GFX/Items/Firstaid/firstaid.fbx";
+		modelSkin = "SCPCB/GFX/Items/Firstaid/firstaid_blue.jpg";
+		modelScale = 0.3;
+
+		pickSoundID=1;
+
+		useModelIcon=true;
+		iconImage = "SCPCB/GFX/Items/Firstaid/inv_firstaid_blue.jpg";
+		iconScale = 0.1;
+		iconRot = Vector3f(-2.3,-0.3,0.2);
+		iconPos = Vector2f(0,0.05);
+		Item::Register(@this);
+	}
+}
+ItemTemplate_BlueFirstAid OriginItem_BlueFirstAid;
+
+
+shared class Item_BlueFirstAid : Item {
+	Item_BlueFirstAid(Item::Template@&in origin) { super(@origin);
+		// onConstructed();
+	}
+
+	void Test() {
+		Debug::log("Im a BlueFirstAid");
+	}
+	void Localize(Item@&in instance) {
+	}
+}
+
+
+
+
+/* Originals --------------------------------------
```

Comment:
Including a copy of the original source code is just noise. If we have to look at it as a reference, we can just go to the original repository.

Reply:
It helps with organization for me at least.

Reply:
This will remain attached to the item until it is imported. It contains essential references conveniently next to the definition, such as strings to be localized, sounds, and heal functions. The issue is many components of the items are scattered, i.e I do not see the sounds in there, hence moving all the code related to that particular item into that item, to help find it. This is actually stored here for refactoring reference purposes. I also note that the other items in OldSrc do not contain much of, if any of this information, also with a lot of items missing. I have moved the other to-be-done items into a folder in the OldSrc folder, and can manage my references there, where they do not apply to a particular data file. Originals may be removed when we are satisfied the item contains all the information desired from the original reference material.